src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -180,6 +180,7 @@ export default Object.freeze({
   ratedDisabilitiesDetectDiscrepancies:
     'rated_disabilities_detect_discrepancies',
   ratedDisabilitiesUseLighthouse: 'rated_disabilities_use_lighthouse',
+  'representativesPortalFrontend': 'representatives_portal_frontend',
   scBrowserMonitoringEnabled: 'sc_browser_monitoring_enabled',
```

Comment:
I believe here only the values are supposed to have the quotes, while keys do not. Not sure if this will break anything, but better to keep it uniform either way.

Reply:
Oops thanks my bad. See commit - c9f304500a9b77ec4f33f823bf53810bb4775f19